macros/utils/table_operations/insert_rows.sql

```diff
@@ -98,7 +98,9 @@
       {% do rendered_column_values.append(column_value) %}
     {% else %}
       {% set column_value = elementary.insensitive_get_dict_value(row, column.name) %}
-      {% do rendered_column_values.append(elementary.render_value(column_value)) %}
+      {% set normalized_data_type = elementary.normalize_data_type(column.dtype) %}
```

Comment:
Can you please verify if any Athena-specific fields need to be added to `data_type_list.sql`? (or if the default implementation is enough?)
To make sure `normalize_data_type` works correctly.

Reply:
ok

Reply:
The default was quite close, but a couple of types were missing. Added `athena__data_type_list` based on https://docs.aws.amazon.com/athena/latest/ug/data-types.html